tools/changelog.main.kts

```diff
@@ -87,8 +87,10 @@ val argsKeyToValue = args
     .filter { it.contains("=") }
     .associate { it.substringBefore("=") to it.substringAfter("=") }
 
-val token = argsKeyToValue["token"] ?:
+val token = argsKeyToValue["token"]
+if (token != null) {
```

Comment:
is it what you want? here was a null check before

Reply:
Yes.

If we write:
```
val token = null ?: println()
```

the token will be `Unit` and be passed as `Unit` in the http headers.